client/client.go

```diff
@@ -171,6 +176,7 @@ func (c *Client) loop() {
 		d := websocket.Dialer{
 			ReadBufferSize:  1024,
 			WriteBufferSize: 1024,
+			HandshakeTimeout: b.Duration(),
```

Comment:
What duration is this intended to be? `b.Duration` increments the backoff timer, though it's already being incremented when there's an error.

Reply:
I use the same backoff timer for convenient. And logic is ok. Should we add another backoff timer to control `HandshakeTimeout`?